tests/test_cli.py

```diff
@@ -2,14 +2,21 @@
 import tempfile
 from unittest.mock import patch
 
+import pytest
+
 from markdown_it.cli import parse
 
 
 def test_parse():
     with tempfile.TemporaryDirectory() as tempdir:
         path = pathlib.Path(tempdir).joinpath("test.md")
         path.write_text("a b c")
-        assert parse.main([str(path)])
+        assert parse.main([str(path)]) == 0
+
+
+def test_parse_fail():
+    with pytest.raises(SystemExit):
+        assert parse.main(["/tmp/nonexistant_path/for_cli_test.md"]) == 1
```

Comment:
nitpick, but surely you will never actually test this assert, because the parse raises a `SystemExit`

Reply:
Yeah, but it's helpful to document that the returncode when SystemExit is raised is 1.

Is there branch coverage on the tests/?

Reply:
I've removed the assert.

https://docs.python.org/3/library/sys.html#sys.exit
https://docs.python.org/3/library/exceptions.html#SystemExit